manifests/init.pp

```diff
@@ -13,7 +13,6 @@
   $ssh_config_hash_known_hosts         = 'USE_DEFAULTS',
   $ssh_config_path                     = '/etc/ssh/ssh_config',
   $ssh_config_owner                    = 'root',
-  $ssh_config_group                    = 'root',
```

Comment:
Hm, on FreeBSD platform, default root group named as 'wheel'. Maybe I missed something - how to avoid reassigning the variable on FreeBSD ?